src/main/java/com/google/cloud/tools/appengine/cloudsdk/internal/args/DevAppServerArgs.java

```diff
@@ -71,4 +72,12 @@
     }
     return Collections.emptyList();
   }
+
+  /**
+   * @return {@code [--name, key1=val1, --name, key2=val2, ...]} or {@code []} if
+   *      keyValues=empty/null.
+   */
+  public static List<String> get(String name, Map<String, String> keyValues) {
+    return Args.namedKeyValues(name, keyValues);
```

Comment:
I was thinking the Args class is the right place for this. I know its just a pass through for this class, but this seems to be the pattern we are following (plus its possible that a gcloud arg may need this in the future).